### PR TITLE
Integrate the service API documents to the GUI

### PR DESCRIPTION
**Pull request by WangLiNaruto**
fixed eclipse-xpanse/xpanse#262
image
image

### CODE REVIEW

File: src/xpanse-api/generated/apis/ServiceApi.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class ServiceApiRequestFactory extends BaseAPIRequestFactory {
         }
 
         // Path Params
-        const localVarPath = '/xpanse/service/openapi/{id}'.replace('{' + 'id' + '}', encodeURIComponent(String(id)));
+        const localVarPath = '/xpanse/register/openapi/{id}'.replace('{' + 'id' + '}', encodeURIComponent(String(id)));
```
Comment (swaroopar) [inline src/xpanse-api/generated/apis/ServiceApi.ts:99]: this file must be auto-generated. No manual changes must be done.

File: src/components/content/order/OrderSubmit.tsx [hunk 2]
```diff
@@ -201,6 +201,25 @@ function OrderSubmit(props: OrderSubmitProps): JSX.Element {
         .concat('&serviceName=', props.name)
         .concat('&latestVersion=', props.version);
 
+    function onclick() {
```
Comment (swaroopar) [inline src/components/content/order/OrderSubmit.tsx:204]: As far as i know, these API changes are not yet final. Xinsui is still working on it.

File: src/components/content/order/OrderSubmit.tsx [hunk 3]
```diff
@@ -209,6 +228,10 @@ function OrderSubmit(props: OrderSubmitProps): JSX.Element {
                 <div className={'services-content'}>
                     <div className={'content-title'}>
                         Service: {props.name}@{props.version}
+                        <button className={'content-title-api'} onClick={onclick}>
+                            API DOCUMENT
```
Comment (swaroopar) [inline src/components/content/order/OrderSubmit.tsx:232]: Text can be `API Documentation`
And please also try other icons - <ApiOutlined />, <LinkOutlined />,

**Review by swaroopar — CHANGES_REQUESTED** [review]
comments added.

File: src/xpanse-api/generated/apis/ServiceApi.ts (hunk 1 above)
Comment (WangLiNaruto) [reply src/xpanse-api/generated/apis/ServiceApi.ts:99]: ok,I'm going to regenerate these files

File: src/components/content/order/OrderSubmit.tsx (hunk 2 above)
Comment (WangLiNaruto) [reply src/components/content/order/OrderSubmit.tsx:204]: I had a discussion with Xinsui just now, he is working, it has no effect on my work content

File: src/components/content/order/OrderSubmit.tsx (hunk 3 above)
Comment (WangLiNaruto) [reply src/components/content/order/OrderSubmit.tsx:232]: OK，I will modify

File: src/xpanse-api/generated/apis/ServiceApi.ts (hunk 1 above)
Comment (WangLiNaruto) [reply src/xpanse-api/generated/apis/ServiceApi.ts:99]: I discussed with xinsui just now, he is modifying the interface and entity, after he finishes modifying, I will regenerate these files

**Comment by swaroopar** [comment]
@WangLiNaruto 
- We must move this link to end of that block.
- We can remove the underline. It is not looking good.
-  Move the icon to the beginning of "API Documentation"
- Also we can reduce the font size.
- Looking at the page size.. may by we can just call the link "API Doc"

**Comment by WangLiNaruto** [comment]
@swaroopar ok,just update,Please take look again

**Review by swaroopar — CHANGES_REQUESTED** [review]
API Documentation link is still not moved to the right end of that block.

**Review by swaroopar — APPROVED** [review]
looks good

**Pull request merged**